.github/workflows/backward-compat-tests.yml

```diff
@@ -49,10 +49,17 @@ jobs:
         with:
           java-version: 1.8
       - name: Build
-        run: ./gradlew stream:server:build -x test ${GRADLE_ARGS}
+        run: mvn -B -nsu -am -pl :stream-storage-server test
```

Comment:
this is not the same thing.
in Gradle "-x test" means to skip tests and not to run tests

Reply:
@eolivelli PTAL again